src/k8s/fa_configmaps.tf

```diff
@@ -21,3 +21,17 @@ resource "kubernetes_config_map" "fa-jvm" {
     TZ = "Europe/Rome"
   }
 }
+
+
+resource "kubernetes_config_map" "fa-ms-customer" {
+  metadata {
+    name      = "famscustomer"
+    namespace = kubernetes_namespace.fa.metadata[0].name
+  }
+
+  data = merge({
+    POSTGRES_SCHEMA = "fa_customer"
+    TZ = "Europe/Rome"
```

Comment:
in others `bpd`/`rtd` ms we used a configmap ref jvm (https://github.com/pagopa/bpd-ms-winning-transaction/blob/release-prod/manifests/pagoPa/deployment-prod.yml#L37) that is missing for `fa-ms-customer `https://github.com/pagopa/fa-ms-customer/blob/release-dev/manifests/pagoPa/deployment-dev.yml)

In my opinion, if there isn't a new requiriment/improvement it's better to use a standard model

Also can we add all possibile variables with the default value? in this way on main we have the list of all possibile variables and their values for a quick and dirty deploy

Reply:
@pasqualedevita Could I consider this comment resolved and try to apply the PR in dev?